### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by ysard**
Night commits...
- Settings for URLs OK #40
- Menu for InfoWidget done (it could be moved on the main view later) #40
- CSV export #31

There are built-in urls in commons.py
and user urls stored in QSettings.

If the user chooses the same name, it should be ok (The QStringList allows non unique elements).

Maybe we could show the template in settings instead of just the name of the database...
=> low priority

Can you take a look at the auto-adjust method for the size of page_box in ViewQueryWidget:updateInfo ?

Known Bug: The CSV export, only exports the variants of the current page;
How can i get all variants of the query without trigger the update of the view ?

### CODE REVIEW

**Comment by ysard** [comment]
Bug discrovered: When the list of urls is changed in settings, the menu is not updated.
We could: 
- reconstruct the menu on every right clicks (bad)
- dispatch a signal when settings are changed, this signal should be received by the plugins (better)
Could you implement this ?

File: cutevariant/gui/viewquerywidget.py [hunk 1]
```diff
@@ -323,7 +341,9 @@ def query(self, query: Query):
 
     def updateInfo(self):
 
-        self.page_info.setText(f"{self.model.total} variant(s)")
+        self.page_info.setText(
+            self.tr("{}-{} of {} variant(s)").format(*self.model.displayed())
```
Comment (dridk) [inline cutevariant/gui/viewquerywidget.py]: It's not readable to have "0-50 of 73" between page selector. And I plan to make the page number editable. 
We can move this information somewhere else in the bar. ( Left side ) 
For example:

     50 shown of 73 variants      <  3 of 40 >

Comment (ysard) [reply cutevariant/gui/viewquerywidget.py]: Ok i will change that.
In fact actual inspiration was SQLite Database Browser

Comment (dridk) [reply cutevariant/gui/viewquerywidget.py]: I tested it ! And finally it's readable ! Thanks for that!  We will see later how to manage page selection

**Pull request merged**